pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools >= 40.6.0",
+    "Cython>=0.29.2",
+    "numpy==1.13.3; python_version=='3.5' and platform_system!='AIX'",
```

Comment:
I looked around a bit more and found this even nicer solution:

https://github.com/scipy/oldest-supported-numpy